frigate/detectors/plugins/openvino.py

```diff
@@ -133,3 +133,68 @@ def detect_raw(self, tensor_input):
                 else:
                     break
             return detections
+        elif self.ov_model_type == ModelTypeEnum.yolov8:
+            out_tensor = infer_request.get_output_tensor()
+            results = out_tensor.data[0]
+            output_data = np.transpose(results)
+            scores = np.max(output_data[:, 4:], axis=1)
+            if len(scores) == 0:
+                return np.zeros((20, 6), np.float32)
+            scores = np.expand_dims(scores, axis=1)
+            # add scores to the last column
+            dets = np.concatenate((output_data, scores), axis=1)
+            # filter out lines with scores below threshold
+            dets = dets[dets[:, -1] > 0.5, :]
```

Comment:
Any reason why this threshold is at 0.5?  This is different than the ssd and yolox models.

Reply:
Performance reasons. YOLOv5 and v8 model output dimensions are usually on the order of thousands to tens of thousands arrays, depending on the model dimensions. So keeping the threshold a little higher showed significant performance improvements.

As an example, YOLOv8 inference+post processing takes ~33ms on my system with threshold of 0.5, while reducing the threshold to 0.3 increased the inference+post processing to ~45ms.

Ideally, we would limit to whatever `min_score` user has in their config, but each camera/area can have a different `min_score` setting, so I haven't thought of a good way to make use of that.